### PR TITLE
feat(android): logcat integration

### PR DESCRIPTION
**Pull request by buenaflor**
Docs for the logcat integration added through SAGP

### CODE REVIEW

**Review by shanamatthews — APPROVED** [review]
minor phrasing tweaks

File: src/platforms/android/configuration/integrations/logcat.mdx [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+      enabled.set(true)
+      minLevel(LogcatLevel.WARNING)
+    }
+  }
+}
+```
+
+## Verify
+
+This snippet captures an intentional error, so you can test that everything is working as soon as you set it up:
```
Comment (shanamatthews) [inline src/platforms/android/configuration/integrations/logcat.mdx:97]:
```suggestion
This snippet captures an intentional error, so you can test that everything is working once you've set it up:
```

File: src/platforms/android/configuration/integrations/logcat.mdx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+<Note>
+
+Supported in Sentry's Android SDK version `6.17.0` and above.
+
+Supported in Sentry Android Gradle Plugin version `3.5.0` and above.
+
+</Note>
+
+The Sentry Logcat integration adds support for automatically attaching breadcrumbs to events above a minimum logging level. Only explicit log calls are captured as breadcrumbs, logs that were created by any third party aren't captured.
```
Comment (markushi) [inline src/platforms/android/configuration/integrations/logcat.mdx:18]: I think we can phrase this more positive.

```suggestion
The Sentry Logcat integration adds support for automatically reporting log calls above a minimum logging level as breadcrumbs. Any matching `android.Log.*` calls inside your app package are captured as breadcrumbs, including `android.Log.*` calls originating from bundled third party libraries.
```

File: src/platforms/android/configuration/integrations/logcat.mdx [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+## Auto-Installation With the Sentry Android Gradle Plugin
```
Comment (markushi) [inline src/platforms/android/configuration/integrations/logcat.mdx:20]:
```suggestion
## Installation With the Sentry Android Gradle Plugin
```

File: src/platforms/android/configuration/integrations/logcat.mdx [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+}
+
+dependencies {
+  implementation("io.sentry:sentry-android:{{ packages.version('sentry.java.android', '6.17.0') }}")
+}
+```
+
+### Configure
+
+Auto-instrumentation is enabled by default, so no further configuration is required. If you'd like to disable the Logcat instrumentation feature, we expose a configuration option for that:
```
Comment (markushi) [inline src/platforms/android/configuration/integrations/logcat.mdx:60]:
```suggestion
Auto-instrumentation is enabled by default, so no further configuration is required. If you'd like to disable the Logcat instrumentation feature, or change the minimum log level the following options are provided:
```

File: src/platforms/android/configuration/integrations/logcat.mdx [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+
+The Sentry Logcat integration adds support for automatically attaching breadcrumbs to events above a minimum logging level. Only explicit log calls are captured as breadcrumbs, logs that were created by any third party aren't captured.
+
+## Auto-Installation With the Sentry Android Gradle Plugin
+
+### Install
+
+To use the Logcat integration, add the Sentry Android Gradle plugin and the Sentry Android SDK (version `6.17.0` or above) in `build.gradle`:
+
+```groovy
```
Comment (markushi) [inline src/platforms/android/configuration/integrations/logcat.mdx:26]:
```suggestion
```groovy {filename:app/build.gradle}
```

File: src/platforms/android/configuration/integrations/logcat.mdx [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+plugins {
+  id "io.sentry.android.gradle" version "{{ packages.version('sentry.java.android.gradle-plugin', '3.5.0') }}"
+}
+
+dependencies {
+  implementation 'io.sentry:sentry-android:{{ packages.version('sentry.java.android', '6.17.0') }}'
+}
+```
+
+```kotlin
```
Comment (markushi) [inline src/platforms/android/configuration/integrations/logcat.mdx:42]:
```suggestion
```kotlin {filename:app/build.gradle.kts}
```

File: src/platforms/android/configuration/integrations/logcat.mdx [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+<Note>
+
+Supported in Sentry's Android SDK version `6.17.0` and above.
+
+Supported in Sentry Android Gradle Plugin version `3.5.0` and above.
+
+</Note>
+
+The Sentry Logcat integration adds support for automatically attaching breadcrumbs to events above a minimum logging level. Only explicit log calls are captured as breadcrumbs, logs that were created by any third party aren't captured.
+
```
Comment (markushi) [inline src/platforms/android/configuration/integrations/logcat.mdx:19]: Could you add a screenshot similar as well here? Here's how I did it for Jetpack Compose: https://github.com/getsentry/sentry-docs/pull/6510/files

File: src/platforms/android/configuration/integrations/logcat.mdx [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+    Sentry.captureException(Exception("My Exception"))
+  }
+}
+```
+
+```java
+import androidx.appcompat.app.AppCompatActivity;
+import android.os.Bundle;
+import java.lang.Exception;
+import timber.log.Timber;
```
Comment (markushi) [inline src/platforms/android/configuration/integrations/logcat.mdx:119]: I think that's the wrong import

File: src/platforms/android/configuration/integrations/logcat.mdx [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+```kotlin
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import java.lang.Exception
+import io.sentry.Sentry
+
+class MyActivity : AppCompatActivity() {
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    Log.w("MyTag", "Warning message.")
```
Comment (markushi) [inline src/platforms/android/configuration/integrations/logcat.mdx:108]: I think the import is missing for `Log`

**Review by markushi — CHANGES_REQUESTED** [review]
Nice one, please have a look at my comments. Also: Let's make sure we merge this after the feature is released.

**Review by markushi — APPROVED** [review]
Nice one, let's ship this!

**Pull request merged**